PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -439,16 +443,45 @@ extension SavesContainerViewController {
             }
         }.store(in: &readableSubscriptions)
 
+        // MARK: Story Presentation
         collection.$presentedStoryWebReaderURL.receive(on: DispatchQueue.main).sink { [weak self] url in
             self?.present(url: url)
         }.store(in: &readableSubscriptions)
 
+        collection.$sharedStoryActivity.receive(on: DispatchQueue.main).sink { [weak self] activity in
+            self?.present(activity: activity)
+        }.store(in: &readableSubscriptions)
+
+        collection.$selectedStoryToReport.receive(on: DispatchQueue.main).sink { [weak self] item in
+            self?.report(item?.givenURL)
+        }.store(in: &readableSubscriptions)
+
         navigationController?.pushViewController(
             CollectionViewController(model: collection),
             animated: true
         )
     }
 
+    private func report(_ givenURL: String?) {
+        guard let givenURL else {
+            Log.capture(message: "Unable to report item from Saves")
+            return
+        }
+
+        let host = ReportRecommendationHostingController(
+            givenURL: givenURL,
+            tracker: model.tracker.childTracker(hosting: .reportDialog),
+            onDismiss: { }
+        )
+
+        host.modalPresentationStyle = .formSheet
+        guard let presentedVC = self.presentedViewController else {
```

Comment:
nit: I think acronyms like `VC` are generally less readable. Just a matter of preference.

Reply:
I agree with this preference as well, updated!